kgforge/specializations/stores/bluebrain_nexus.py

```diff
@@ -391,11 +397,13 @@ def _download_many(
         paths: List[str],
         store_metadata: Optional[DictWrapper],
         cross_bucket: bool,
+        content_type: str
     ) -> None:
         async def _bulk():
             loop = asyncio.get_event_loop()
             semaphore = Semaphore(self.service.max_connection)
-            async with ClientSession(headers=self.service.headers_download) as session:
+            headers = update_dict(self.service.headers_download, {"Accept":content_type})
```

Comment:
This should only be executed when `content_type` is NOT `None`. Because the `forge.download` method passes `None` as default. So it gives the error `request is missing required HTTP header 'Accept'` when `content_type` is not given

Reply:
This is fixed